### ISSUE TITLE
runAction arguments contain `DBusString`

### ISSUE THREAD

**Issue by aquinary** [Merrit/krunner-dart#2]
In 
```dart
Future<void> runAction({required String matchId, String? actionId})
```
`matchId` and `actionId` contains `DBusString(%string%)`.

I have to do this:
```dart
matchId.replaceAll('DBusString(', '').replaceAll(')', '');
```

**Comment by Merrit** [comment]
Can confirm, thanks for opening the issue! I'll take a look at it.

**Comment by Merrit** [comment]
Should be resolved with 1.0.1

Thanks again for opening the issue! :blue_heart:

**Comment by aquinary** [comment]
I am glad that dart is moving beyond flutter as well. I use it very actively in my projects. And I was pleasantly surprised that the plugin for krunner can be written in a language that is pleasant to me. Thank you.
